internal/repository/oci/export_test.go

```diff
@@ -0,0 +1,6 @@
+package oci
+
+// compiled only when running tests.
```

Comment:
Why introduce this indirection and not create a separate pkg for the wrapper isolated with its tests.
And then inject the wrapper dependency for consumers as usual.

Reply:
Why introduce another package "B" just to **test** package "A"?
Packages should have purpose. In this case, package "A" - `internal/repository/oci` can "handle itself" very well, we can write all the tests we want without introducing new abstractions.
I guess "do not multiply entities beyond necessity" is still the correct approach?